### PR TITLE
KNX: `local_ip` for routing is optional

### PR DESCRIPTION
**Pull request by farmio**
## Proposed change

`local_ip` for routing is optional

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46133
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -49,7 +49,7 @@ There is currently support for the following device types within Home Assistant:
 To use your KNX bus in your installation, add the following lines to your `configuration.yaml` file:
 
 ```yaml
-knx:
+knx: ~
```
Comment (frenck) [inline source/_integrations/knx.markdown]: Our styling guidelines prefer the implicit use of a nulled value.

```suggestion
knx:
```

File: source/_integrations/knx.markdown [hunk 2]
```diff
@@ -142,15 +142,14 @@ Explicit connection via KNX/IP routing. This requires multicast communication to
 
 ```yaml
 knx:
-  routing:
-     local_ip: "192.168.2.109"
+  routing: ~
```
Comment (frenck) [inline source/_integrations/knx.markdown]: Our styling guidelines prefer the implicit use of a nulled value.

```suggestion
knx:
```

**Review by frenck — CHANGES_REQUESTED** [review]
This PR targets the `current` branch, which is incorrect. As this PR depends on a change in the code base, it depends on a next release of Home Assistant Core. Thus it needs to target the `next` branch.

Please change the target branch of this PR and git rebase this PR if needed. Thanks 👍

File: source/_integrations/knx.markdown [hunk 3]
```diff
@@ -142,15 +142,14 @@ Explicit connection via KNX/IP routing. This requires multicast communication to
...
 ```
 
 {% configuration %}
 local_ip:
   description: The local IP address of the interface that shall be used to send multicast packets.
```
Comment (farmio) [inline source/_integrations/knx.markdown]:
```suggestion
  description: The local IP address of the interface that shall be used to send multicast packets. If omitted the default multicast interface is used.
```

File: source/_integrations/knx.markdown [hunk 4]
```diff
@@ -142,15 +142,14 @@ Explicit connection via KNX/IP routing. This requires multicast communication to
 
 ```yaml
 knx:
-  routing:
-     local_ip: "192.168.2.109"
+knx:
```
Comment (frenck) [inline source/_integrations/knx.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**